app/src/main/java/com/asfoundation/wallet/topup/payment/PaymentAuthPresenter.java

```diff
@@ -249,14 +249,13 @@ private void handleAdyenUriResult() {
         }, this::showError));
   }
 
-  private void handleAdyenUriRedirect(String domain, String amount, String transactionType) {
+  private void handleAdyenUriRedirect() {
     disposables.add(adyen.getRedirectUrl()
         .observeOn(viewScheduler)
         .filter(s -> !waitingResult)
         .doOnSuccess(redirectUrl -> {
           view.showLoading();
-          navigator.navigateToUriForResult(redirectUrl, billingService.getTransactionUid(), domain,
-              "", new BigDecimal(amount), transactionType);
+          navigator.navigateToUriForResult(redirectUrl);
           waitingResult = true;
         })
         .subscribe(__ -> {
```

Comment:
- onError method can be replaced with method reference.

- Please look at all the methods, the same applies for almost all of them